### PR TITLE
fix: save image timeout 2

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Part of FL-1725

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: charts/substra-backend/templates/statefulset-builder.yaml [hunk 1]
```diff
@@ -200,6 +200,8 @@ spec:
             value: {{ toYaml .Values.builder.kanikoStartup.maxAttempts | quote }}
           - name: BUILDER_KANIKO_STARTUP_PENDING_STATE_WAIT_SECONDS
             value: {{ toYaml .Values.builder.kanikoStartup.checkDelay | quote }}
+          - name: IMAGE_SAVING_TIMEOUT_SECONDS
+            value: {{ .Values.builder.saveImageTimeoutSeconds | quote }}
```
Comment (thbcmlowk) [inline charts/substra-backend/templates/statefulset-builder.yaml:204]: Is it needed for the `builder` itself?

File: examples/values/backend-org-1.yaml [hunk 2]
```diff
@@ -131,3 +131,6 @@ orchestrator:
     - yourchannel:
         restricted: false
         model_export_enabled: true
+
+builder:
+  saveImageTimeoutSeconds: 180
```
Comment (thbcmlowk) [inline examples/values/backend-org-1.yaml:136]: I'm not sure that was a problem in the CI, was it?

**Review by thbcmlowk — COMMENTED** [review]
Should we also update the main default value?

File: examples/values/backend-org-1.yaml (hunk 2 above)
Comment (guilhem-barthes) [reply examples/values/backend-org-1.yaml]: Good catch, thanks

File: charts/substra-backend/templates/statefulset-builder.yaml (hunk 1 above)
Comment (guilhem-barthes) [reply charts/substra-backend/templates/statefulset-builder.yaml:204]: In my testing, in some cases the  worker was not sufficient, will retest

Comment (thbcmlowk) [reply charts/substra-backend/templates/statefulset-builder.yaml:204]: Also confirmed on my end, LGTM

**Review by thbcmlowk — APPROVED** [review]
LGTM, thanks for fixing

**Pull request merged**